tests/HomePage.spec.js

```diff
@@ -1,69 +1,66 @@
-const chai = require("chai");
-const expect = chai.expect;
+const { expect } = require("chai");
+const cardsNames = require("../objects/homePage");
 const App = require("../test/pageobjects/App");
 const HomePage = require("../test/pageobjects/HomePage");
 
 describe("Check Home Page smoke tests functionality", () => {
-  it("Check if page opens and ToolsQA logo appears", async () => {
+  beforeEach(async () => {
     await App.openHomePage();
+    if (await App.ad.isDisplayed()) {
+      await App.removeAd();
+    }
+  });
+  it("Check if page opens and ToolsQA logo appears", async () => {
     await (await HomePage.logoImg).waitForDisplayed();
-    await expect(await (await HomePage.logoImg).isDisplayed()).to.be.true;
+    expect(await (await HomePage.logoImg).isDisplayed()).to.be.true;
   });
   it("Check if all the six entities can be viewed on Home Page", async () => {
-    const elementsCard = await (await HomePage.elementsCard).getText();
-    await expect(await elementsCard).contains("Elements");
-
-    const formsCard = await (await HomePage.formsCard).getText();
-    await expect(await formsCard).contains("Forms");
-
-    const alertsCard = await (await HomePage.alertsCard).getText();
-    await expect(await alertsCard).contains("Alerts, Frame & Windows");
-
-    const widgetsCard = await (await HomePage.widgetsCard).getText();
-    await expect(await widgetsCard).contains("Widgets");
-
-    const interactionsCard = await (await HomePage.interactionsCard).getText();
-    await expect(await interactionsCard).contains("Interactions");
-
-    const bookStoreCard = await (await HomePage.bookStoreCard).getText();
-    await expect(await bookStoreCard).contains("Book Store Application");
+    const cards = [
+      "Elements",
+      "Forms",
+      "Alerts, Frame & Windows",
+      "Widgets",
+      "Interactions",
+      "Book Store Application",
+    ];
```

Comment:
You have the cards defined in the objects/homePage.js file, import that object and use it here.